kura/org.eclipse.kura.asset.cloudlet.provider/OSGI-INF/AssetCloudletComponent.xml

```diff
@@ -1,16 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-    Copyright (c) 2016 Eurotech and/or its affiliates and others
+    Copyright (c) 2016, 2017 Eurotech and/or its affiliates and others
 
      All rights reserved. This program and the accompanying materials
      are made available under the terms of the Eclipse Public License v1.0
      which accompanies this distribution, and is available at
      http://www.eclipse.org/legal/epl-v10.html
 -->
-<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" 
-	activate="activate" 
-	deactivate="deactivate" 
-	name="org.eclipse.kura.asset.cloudlet">
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" deactivate="deactivate" immediate="true" name="org.eclipse.kura.asset.cloudlet">
```

Comment:
I've setup a wire asset in a develop build and, as you can see in the following images, the cloudlet is not registered and cannot be invoked from remote.

![schermata 2017-03-30 alle 13 53 20](https://cloud.githubusercontent.com/assets/10448126/24503551/db3be274-1552-11e7-9730-5be56782b91b.png)

<img width="1327" alt="schermata 2017-03-30 alle 13 52 13" src="https://cloud.githubusercontent.com/assets/10448126/24503545/d39bf626-1552-11e7-8de5-7e06cb8e6033.png">

This is, instead, what you get when you compile the branch of the suggested PR:
![schermata 2017-03-30 alle 14 08 55](https://cloud.githubusercontent.com/assets/10448126/24503575/eed81118-1552-11e7-97a0-65896b363f07.png)
<img width="1295" alt="schermata 2017-03-30 alle 14 09 43" src="https://cloud.githubusercontent.com/assets/10448126/24503583/f4b2eafe-1552-11e7-856f-9f22b5606a3e.png">

As you can see, in the first case, the ASSET-V1 cloudlet is not started and registered.
So, maybe I've missed something, but I think that with the current implementation, the cloudlet should be started immediately to have it working.